### ISSUE TITLE
missing `=` for python

### ISSUE THREAD

**Issue by nerohmot** [Semi-ATE/maxiconda-bot#8]
In the `solutions` on maxiconda-envs, there is a missing equal sign between python and the version number.

**Comment by goanpeca** [comment]
Should we turn the format into a json?

**Comment by nerohmot** [comment]
makes sense ... didn't the prototype already use json under the hood ? I seem to remember that I used it ...

**Comment by nerohmot** [comment]
Oh, now I remember, it is not so easy to read (I wanted to split python / primary packages / secondary packages) and in .json it is not controllable (I even think alphabetically ordered) ... 
Anyway, json is better, for the users there will be the `consolidate.py`, there we can organize the split for easy reading :-)

**Comment by goanpeca** [comment]
I mean we can create our own json file that stores the infomation we need in a way we can use it, but preserving as much of the original dry-sole as possible.

**Comment by goanpeca** [comment]
Should we have separate consolidated files for each new version of the metapackage? maybe a folder for the xls. tables
